.github/workflows/user.yaml

```diff
@@ -2,67 +2,67 @@
 name: Deploy user microservice
 
 on:
-    # Deploys the microservice on push to subfolder user on the main branch of this code repository.
-    push:
-        branches:
-            - main
-        paths:
```

Comment:
Seems like we have different tab size settings. Mine is set to tab size: 4. Do you have 2 perhaps?

Reply:
I don't know hove to change the settings for yaml files they are not formatted as the rest of my files sadly. also have tab size 4.

Reply:
Tried to fix it but I can't...

Reply:
Thats okay.